pimd/pim6_cmd.h

```diff
@@ -39,14 +39,5 @@
 #define DEBUG_MLD_TRACE_STR "MLD internal daemon activity\n"
 
 void pim_cmd_init(void);
-/*
```

Comment:
The branch is not rebase to master, why this file is present?
Please rebase, might we get some conflicts